### ISSUE TITLE
[AllyAudit] Link text does not clearly indicate the purpose or destination of the link [I]

### ISSUE THREAD

**Issue by ssabo1** [sparkdesignsystem/Backlog#1328]
**User Story**
As Spark we want to 

**Notes:**
Paciello Audit on Spark Design System
QLC-20 Promo
Insufficient link text

ISSUE DESCRIPTION
When link text does not clearly indicate the purpose or destination of the link, and surrounding text does not provide supporting context, some people may find it difficult or impossible to understand the link's functionality, leading to reduced browsing efficiency. For example, screen reader users may be using their screen reader's ability to list links on a page, but if link text is ambiguous or vague, browsing efficiency is reduced. People with cognitive impairments that affect their ability to process text may also find it difficult to understand the link's purpose.

On 85 - Promo, 87 - Promo (image left), and 88 - Promo (image right), there is a "Learn More" link that is not enclosed within a sentence or paragraph to provide context. This forces screen reader users to explore the surrounding content to gain additional context.

RECOMMENDATIONS
Generic link phrases, such as "Learn More" do not provide sufficient context as to the purpose of the link. When there are multiple links of this nature on a page, it may be difficult to tell them apart. Instead, reword the link text so that the destination of the link is conveyed by the link text, such as “Learn more about the company”. Where this is not feasible or desirable, provide additional information on the purpose of the link - for instance, using the aria-label attribute or using visually hidden text that will still be announced by screen readers.

**AC**

**Comment by ssabo1** [comment]
confirm, No longer an issue?

**Comment by ssabo1** [comment]
Do we want to do this as a Design System???

**Comment by ssabo1** [comment]
Confirmed: This has been added to Spark Design System Team Backlog.
